### DISCUSSION TITLE
How can we check view blade file for article crud

### DISCUSSION THREAD

**Discussion by shilpitrusty** [Laravel-Backpack/community-forum#916] [General] [answered]
How can we check view blade file for article crud? I didn't find blade template file for article crud, category crud, tag crud. I want to add new field in article crud

**Comment by pxpm** [comment, marked as answer]
Hey @shilpitrusty 

This is what you are looking for:  https://github.com/Laravel-Backpack/NewsCRUD?tab=readme-ov-file#overwriting-functionality

Don't forget you also need to change the models to add the new fields in the `$fillable`, and also change the migrations to accomodate your new fields. 

By that time, I think it's better to just copy to package controllers/models/routes into your application and just uninstall the package. 

Cheers
